backend/tet/events/linkedevents.py

```diff
@@ -65,11 +66,13 @@ def _api_call(
                 # thus we don't try to get the response body
                 # TODO in some cases it would be helpful to inspect this body to find
                 # where the error originates (a forgotten slash has been seen while testing)
+                # print('error is: ' + e.response.text)
                 raise LinkedEventsException(
                     code=503, detail="Server error from Linked Events."
                 )
             # TODO 403 is actually configuration error in Linked Events for the TET API key
             elif e.response.status_code == 401 or e.response.status_code == 403:
+                # print('error is: ' + e.response.text)
```

Comment:
I would not include this commit because putting code in comments is normally not
good code practice. If the code should be included then it shouldn't be in comments
and if it shouldn't be included then it should be deleted. There are exceptions as to
almost every case but I don't think this is one of those. What do you think?

Reply:
it can be removed. I just spent at least half an hour to find a way to print the whole error message. `e` or `e.response` didnt print the message. but perhaps "real" backend developers know instantly how to print or debug the error :)

Reply:
This commit is now deleted